src/bioregistry/data/contexts.json

```
@@ -13,7 +13,12 @@
         "github": "matentzn",
         "name": "Nico Matentzoglu",
         "orcid": "0000-0002-7356-1779"
-      }
+      },
+      {
+        "github": "anitacaron",
+        "name": "Anita Caron",
+        "orcid": "0000-0002-6523-4866"
+      },
```

```suggestion
      }
```

Suggestion from @gouttegd